src/main/kotlin/me/spradling/gift/core/api/RestVerticle.kt

```diff
@@ -42,8 +43,10 @@ class RestVerticle @Inject constructor(val configuration: GiftCommitConfiguratio
 
     router.route(HttpMethod.POST, "/*").handler(BodyHandlerImpl())
     router.route(HttpMethod.PATCH, "/*").handler(BodyHandlerImpl())
+
     router.route("/swagger/*").handler(StaticHandler.create("swagger"))
     router.get("/health").handler(HealthHandler())
+    router.get("/v1/accounts").handler(GetAccountsHandler(storageClient))
```

Comment:
nit: i don't really care but probably a slightly better name for this would be `ListAccounts` since it is a get all to differentiate the `GetAccount` that is soon to follow.